spec/testdata/tags.yml

```diff
@@ -7,3 +7,7 @@
   name: nyc
   description: "anything related to New York City"
   id: 2
+-
+  name: finance
```

Comment:
i also added a tag here (philly -- because i needed a restricted tag for testing), and also gave it id 3, will make sure to accept this and give philly a different id.

reminds me of a thought that occured to me when i was futzing with tag test data.  don't think it's worth pursuing now, but i think over time it will become confusing if our test data veers too far away from our actual tag data. ("oil" instead of fossil-fuel, etc.) implementing

at a point that makes sense (once production tags have stabilized?) i propose we pick the first 3 (or 4) tags from the actual tag list and use those as the test data to avoid confusion. again: just a thought for later.